server/src/helpers/sendEmail.ts

```diff
@@ -0,0 +1,45 @@
+import nodemailer from 'nodemailer';
+import { config } from 'dotenv';
+
+config();
```

Comment:
as we import `dotenv`, in the app no need to import it everywhere,
As you will run `sendEmail` in the app cycle so that it will be available to me
Just import `SECRET_EMAIL`, `SECRET_PASSWORD` inside `sendEmail`